libs/checkpoint/langgraph/checkpoint/serde/types.py

```diff
@@ -31,13 +28,7 @@ def UpdateType(self) -> Any: ...
 
     def checkpoint(self) -> Optional[C]: ...
 
-    def from_checkpoint(
-        self, checkpoint: Optional[C], config: RunnableConfig
-    ) -> Generator[Self, None, None]: ...
-
-    async def afrom_checkpoint(
```

Comment:
removing async?

Reply:
This was removed from the actual lib a while ago, but had been left in this protocol in checkpoint lib